Errors when downloading FFO data

Goal, 
Run the current FFO manager data.   Get roster data and the combined CSV.

Environment
OS:  Pop!_OS 22.04
Version: ffo_scraper_1.1.0_linux_amd64

Session capture

`dthacker@lab:~/ffomanager/scouting/season6/scraper$ ./ffo_scraper -excel-export

 FFO Player Scraper 
                    
Loading clubs		 ... done!
Scraping rosters         ... done! [80 in 5.182s]
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
player-scraper/internal/core.ExportToCsv({0xc00013e908, 0x50, 0xba1630?}, {0xc00002d680, 0x3c}, 0x1, {0xa8a0fe, 0xc}, {0xa8c16b, 0xf})
	/Users/jameshowe/Desktop/projects-new/esmshub/player-scraper/internal/core/csv_export.go:196 +0x1247
main.main()
	/Users/jameshowe/Desktop/projects-new/esmshub/player-scraper/cmd/ffo/main.go:139 +0xc10`

- Rosters are being downloaded
- CSV is not created 
- roster and wages files are present on the FFO site. 

Let me know if I can send more debug info
Cheers
Dave

--- a/internal/core/csv_roster_parser.go
+++ b/internal/core/csv_roster_parser.go
@@ -13,7 +13,7 @@ func (p *TextRosterParser) Parse(data io.Reader) (*[][]string, error) {
 	rows := [][]string{}
 	for scanner.Scan() {
 		line := scanner.Text()
-		if strings.HasPrefix(line, "---") {
+		if strings.HasPrefix(line, "---") || len(strings.TrimSpace(line)) == 0 {
 			continue
 		}
 

